community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/CallProcedureAcceptanceTest.scala

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.internal.cypher.acceptance
+
+import java.util.stream.Stream
+
+import org.neo4j.cypher.{CypherTypeException, ExecutionEngineFunSuite, InvalidArgumentException}
+import org.neo4j.kernel.api.KernelAPI
+import org.neo4j.proc.Procedure.{BasicProcedure, Context}
+import org.neo4j.proc.ProcedureSignature.procedureSignature
+import org.neo4j.proc.{Neo4jTypes, ProcedureSignature => KernelSignature}
+
+class CallProcedureAcceptanceTest extends ExecutionEngineFunSuite {
```

Comment:
Add a test for calling a non-existent procedure, and tests were argument values are `null`s.